src/OpenTelemetry.Extensions.PersistentStorage/README.md

```diff
@@ -1,12 +1,12 @@
-# Persistent Storage Interface and Implementation
+# Persistent Storage file based implementation
 
 [![NuGet](https://img.shields.io/nuget/v/OpenTelemetry.Extensions.PersistentStorage.svg)](https://www.nuget.org/packages/OpenTelemetry.Extensions.PersistentStorage)
 [![NuGet](https://img.shields.io/nuget/dt/OpenTelemetry.Extensions.PersistentStorage.svg)](https://www.nuget.org/packages/OpenTelemetry.Extensions.PersistentStorage)
 
-This package provides both the interface and implementation of persistent
-storage. It is an experimental component which can be used by OpenTelemetry
-Exporters to provide reliable data delivery. Eventually this component should
-get splitted into the abstract interface and concrete implementations.
+This package provides file based implementation of
+[persistent-storage-abstractions](../OpenTelemetry.Extensions.PersistentStorage.Abstractions/README.md#Persistent-Storage-Abstractions)
+It is an experimental component which can be used by OpenTelemetry Exporters to
+provide reliable data delivery.
```

Comment:
I suspect "reliable data delivery" is undefined and might be an overstatement, perhaps "to improve the reliability of data delivery".